src/stdplus/tab.hpp

```diff
@@ -148,16 +148,19 @@ typename T2::iterator copy_safe_apart(size_t n, const T1 & tab_src, T2 & tab_dst
 		template <typename TC> // TC = type of some container. it must be linear-memory. TODO: metaprogramming check
 		tab_range(TC & obj, size_t begin, size_t end)
 				: m_begin(&(obj.at(begin))), m_end(&(obj.at(end-1))+1), m_size(end-begin) {
-			if (end<begin) throw std::invalid_argument("Incorect begin or end (end<begin).");
+			// std::less not required for size_t, but using it to be like the other code above (and correct for review or copy-paste)
+			if (std::less<size_t>()(end,begin)) throw std::invalid_argument("Incorect begin or end (end<begin).");
```

Comment:
typo "incorrect"
also in other places
@mikurys please fix at some point by the way
:pencil2: